.github/workflows/main.yml

```diff
@@ -48,7 +62,7 @@ jobs:
       - name: Install cargo-llvm-cov
         uses: taiki-e/install-action@cargo-llvm-cov
       - name: Generate code coverage
-        run: cargo llvm-cov --all-features --workspace --lcov --output-path lcov.info
+        run: cargo llvm-cov --bin lndk --workspace --lcov --output-path lcov.info
```

Comment:
Could you explain what this does?

Reply:
With the `--all-features` flag codecov includes the integration tests in the coverage, but I think we only want to be looking at the unit tests. So `--bin lndk` specifies that we only want to be looking at the tests in /src for the lndk binary. Would it be helpful to add a note about this in the commit?